.gitignore

```diff
@@ -1,3 +1,5 @@
 local.properties
 **/pana-report
 **/.pubignore
+
+.DS_Store
```

Comment:
I'd recommend to remove the `.DS_Store` files from the repo, entirely.

Reply:
@bishopb , agree. missed that one. git ignore probably was not configured for it yet...